assets/images/utils/check.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+	stroke-linejoin="round" viewBox="0 0 24 24">
+	<path d="M20 6L9 17l-5-5" />
+</svg>
```

Comment:
Optimize the SVG (like you did in the other PR)

Reply:
I already optimized it, though my editor added the newlines. Some other SVGs also have newlines so I left them in.

Reply:
You don't need the XMLNS attribute, just a nitpick though (since the svgs are injected)

Reply:
I left it in because it's in a fair amount of other SVGs, I can remove it from all of them though.

Reply:
Done in 4c5f116